libs/types/src/fixed_point.rs

```diff
@@ -293,15 +293,16 @@ pub trait FixedPointNumberExtension: FixedPointNumber {
 	/// rounding preference. Returns None if out of bounds.
 	///
 	/// Note:  This assumes that the FP accuracy has been adjusted to match
-	/// the accuracy of the FP extended type in question (Rate in this case).
-	/// For example:
-	/// Rate::saturating_from_rational(2).checked_div_with_rounding(2,
+	/// the accuracy of the FP extended type in question (FixedU128 in this
+	/// case). For example:
+	/// FixedU128::saturating_from_rational(2).checked_div_with_rounding(2,
```

Comment:
I suppose this also needs some tweaking?

Reply:
Why?